runner-dist/apisix-runner-src-dist/pom.xml

```diff
@@ -63,8 +63,8 @@
                         </goals>
                         <configuration>
                             <target>
-                                <copy file="${project.build.directory}/apisix-runner-src.tgz"
-                                      tofile="${project.basedir}/../../dist/apache-apisix-java-plugin-runner-${project.version}-src.tgz"
+                                <copy file="${project.build.directory}/apisix-java-plugin-runner.tgz"
+                                      tofile="${project.basedir}/../../apisix-java-plugin-runner-0.1.0.tgz"
```

Comment:
I recommend replacing the version number with `${project.version}`.
And the version could exclude suffix `-RELEASE`.

Reply:
got

Reply:
update